extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/WikiParquetReaderTest.java

```diff
@@ -59,7 +62,7 @@ public void testWiki() throws IOException
     reader = createReader("example/wiki/wiki.parquet", schema, JSONPathSpec.DEFAULT);
     List<InputRowListPlusRawValues> sampled = sampleAllRows(reader);
 
-    final String expectedJson = "{\n"
+    final String expectedJsonString = "{\n"
```

Comment:
Would it be possible to revert this variable name change - and in line 83 as well? 
It would be great to have as few changes as possible to fix the flaky test.